Update of technical message types

domna opened the pull request:
I did some refinement of the technical messages. The version in the document were outdated and we actually simplified and structured them differently now. @boris-ibl If you could also regenerate the word document with it that would be great.

domna committed:
Update MET-SWIM-AMQP-Guidance.md

domna committed:
Add all technical message types

boris-ibl commented:
Hi Florian, thank you for the update, I merged the pull request. I would like to generate the Word, but still need to apply the various changes that have been discussed on the MET3SG Sharepoint. But if you need an updated Word with the technical messages part only just let me know and I will run it.

domna commented:
I think for now it's fine. I just removed the old parts on the eurocontrol sharepoint and added a link to this documentation here. If you can easily generate just the technical messages part that would be nice, then I can update the word file there. Btw, what tool are you using for generating the file? The output looks pretty neat :)

boris-ibl commented:
I apologise, but I was away from the office for a couple of days. The tool I am using is **vscode-pandoc** extension for VSCode and Cursor, it can transform Markdown into HTML, DOCX, PDF, etc.  I updated the .docx and added it into this repository.
